codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/GoWriter.java

```diff
@@ -269,10 +269,23 @@ public String apply(Object type, String indent) {
             if (type instanceof Symbol) {
                 Symbol typeSymbol = (Symbol) type;
                 addUseImports(typeSymbol);
-                if (isExternalNamespace(typeSymbol.getNamespace())) {
-                    return formatWithNamespace(typeSymbol);
+
+                Boolean isUniverseType = typeSymbol.getProperty(SymbolUtils.GO_UNIVERSE_TYPE, Boolean.class)
```

Comment:
Doesn't matter, but this could probably be `boolean`

Reply:
Got this cleaned up as well.